<ui version="4.0" >
 <class>UIProjectsManager</class>
 <widget class="QDockWidget" name="UIProjectsManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Projects Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>3</number>
    </property>
    <property name="leftMargin" >
     <number>5</number>
    </property>
    <property name="topMargin" >
     <number>5</number>
    </property>
    <property name="rightMargin" >
     <number>5</number>
    </property>
    <property name="bottomMargin" >
     <number>5</number>
    </property>
    <item>
     <widget class="QFrame" name="fToolButtons" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>3</number>
       </property>
       <property name="leftMargin" >
        <number>5</number>
       </property>
       <property name="topMargin" >
        <number>5</number>
       </property>
       <property name="rightMargin" >
        <number>5</number>
       </property>
       <property name="bottomMargin" >
        <number>5</number>
       </property>
       <item>
        <widget class="QToolButton" name="tbOpen" >
         <property name="text" >
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbClose" >
         <property name="text" >
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbSave" >
         <property name="text" >
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbSaveAs" >
         <property name="text" >
          <string>Save As</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbSettings" >
         <property name="text" >
          <string>Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbSources" >
         <property name="text" >
          <string>Sources</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="tvProjects" >
      <property name="editTriggers" >
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
